conclave-host/src/main/kotlin/com/r3/conclave/host/Gramine.kt

```diff
@@ -0,0 +1,109 @@
+package com.r3.conclave.host
```

Comment:
This package is part of the public API namespace, and this being an internal class cannot reside here. Move this to the internal package.